README.md

```diff
@@ -176,11 +189,42 @@ on it.
 
 If you need proper return values take a look at [Riptide: Capture](riptide-capture).
 
-#### Exceptions
+### Exceptions
 
 The only special custom exception you may get is `NoRouteException`, if and only if there was no matching condition and
 no wildcard condition either.
 
+### Plugins
+
+Riptide comes with a way to register extensions in the form of plugins.
+
+- `OriginalStackTracePlugin`, preserves stack traces when executing requests asynchronously
+- [`FailsafePlugin`](riptide-failsafe), adds retries and circuit breaker support
+- [`TransientFaultPlugin`](riptide-faults), detects transient faults, e.g. network issues
+- [`TimeoutPlugin`](riptide-timeout), applies timeouts to the whole call (including retries, network latency, etc.)
+
+Whenever you encounter the need to perform some repetitive task on the futures returned by a remote call,
+you may consider implementing a custom Plugin for it, e.g.:
+
+```java
+class MetricsPlugin implements Plugin {
```

Comment:
maybe add a one-liner to this example on how to register such a plugin (for people too lazy to read APIs, not for me...)

Reply:
Done, even though it's shown earlier in the readme, in general

Reply:
Ah ok missed that, still I like it with it included :wink: